language-microgenerator/samples/language_sentiment_text.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
```

Comment:
Existing sample modified for the microgenerator client: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/language/generated-samples/v1/language_sentiment_text.py